components/motor/ulnstepper/28byj-48_test.go

```diff
@@ -268,7 +268,7 @@ func TestFunctions(t *testing.T) {
 		err = m.GoFor(ctx, 146, 1, nil)
 		test.That(t, err, test.ShouldBeNil)
 		allObs = obs.All()
-		latestLoggedEntry = allObs[len(allObs)-1]
+		latestLoggedEntry = allObs[len(allObs)-2]
```

Comment:
what caused this to change?

Reply:
I added a new logger line to tell the user that we're setting the rpm to max, not stopping, when we're passing in an rpm larger than the maxRPM, it was stopping because the logic previously called a stop when any warning was returned from CheckSpeed, not we only stop when an error is returned - the zero rpm error.

Reply:
So now we check the entry before last to make sure it's the maxrpm warning that we expect.